### ISSUE TITLE
Platform Pyramid (Python) does not record additional context information

### ISSUE THREAD

**Issue by miohtama** [getsentry/sentry#1557]
When one has set *Platform* to *Pyramid (Python)* in the Sentry UI project settings, the additional context information, like HTTP headers, from the client are not coming through. I am not sure if they are not recorded or not displayed.

The workaround is to set *Platform* to *Python*, without Pyramid specifity.

This was tested with [pyramid_raven](https://github.com/thruflo/pyramid_raven) and Sentry 7.4.3.

**Comment by dcramer** [comment]
The platform value on the server wouldn't actually control this. It's possible you were looking at two diff events, one that simply didn't have it?

**Comment by miohtama** [comment]
I had this issue on the same configuration with two different Sentry instance. One worked, the other didn't. I finally nailed it down the differences to this setting and changing it had effect - though not sure if it was a side effect of resaving settings. I'll do some additional testing to confirm it was actually caused by this.

The event is definitely the same, as I caused the same exception on two different servers. The only difference was one was pointing to another Sentry project (on the same Sentry server).
